res/Sectors/M1105/Darret.txt

```diff
@@ -0,0 +1,686 @@
+0702                      C737658-6 Ni                           { -2 } (654+3) [4426] - -  - 924 11 NaXX M6 V      
+0703                      C21048B-6 Ni                           { -2 } (934-4) [2259] - -  - 103 16 NaXX M7 V M0 V       
+0706                      A798442-D Ni Pa                        {  1 } (833-1) [35AF] - -  - 121 14 NaXX F1 IV M7 V       
+0707                      B626689-8 Ni                           { -1 } (B50-1) [351C] - -  - 722 15 NaXX M7 V       
+0709                      B572686-A He Ni                        {  1 } (D54+0) [676D] - -  - 522 12 NaXX G1 IV       
+0710                      C656526-8 Ag Ga Ni                     { -1 } (A41-3) [5489] - -  - 105 15 NaXX M9 V        
+0801                      C10099A-A Hi Na Va In                  {  3 } (B8B-5) [7C75] - -  - 700  9 NaXX F3 V       
+0802                      C8A38DF-7 Fl Ph                        { -1 } (A79+2) [A759] - N  - 103 10 NaXX M7 V M8 V       
+0803                      E697211-5 Lo                           { -3 } (811-4) [1114] - -  - 104 16 NaXX M7 V       
+0804                      C610488-8 Ni                           { -2 } (630+4) [428A] - S  - 402  9 NaXX A6 V K3 V       
+0807                      B464305-A Lo                           {  1 } (621+1) [346A] - N  - 602 10 NaXX M5 V       
+0808                      C9A6676-9 Fl Ni                        { -1 } (D55+1) [853B] - -  A 415 15 NaXX M0 V       
+0810                      C676142-4 Lo                           { -2 } (801+1) [1111] - -  - 203 14 NaXX G0 V       
+0901                      C301735-8 Ic Na Va Pi                  { -1 } (E62-1) [5629] - -  - 613 12 NaXX M9 III G3 V       
+0904                      B413622-9 Ic Na Ni                     {  0 } (555-3) [5649] - -  - 502 13 NaXX M5 V       
+0906                      C62A797-A Pi                           {  1 } (86B-2) [988B] - N  - 913 10 NaXX M5 V M9 V       
+1002                      C334755-6                              { -1 } (864-3) [665A] - S  - 120  6 NaXX M5 V K0 V       
+1003 Borindee             B5398BC-C Ph (Borin)                   {  2 } (57B-4) [9A4F] - -  - 802  7 NaXX F1 V       
+1004                      C648210-6 Lo                           { -2 } (811-1) [1183] - S  - 901 11 NaXX F9 V M9 V       
+1010                      AAB6725-B Fl                           {  2 } (968-3) [696A] - -  - 202  8 NaXX G4 V       
+1101                      D97A100-5 Lo Wa                        { -3 } (901+0) [1159] - S  - 214 12 NaXX M9 V       
+1102                      B897666-8 Ag Ni O:1003                 {  0 } (953+1) [666A] - N  - 603  9 NaXX F0 V       
+1103                      B98A553-A Ni Wa Pr                     {  1 } (F42-3) [864A] - S  - 925 18 NaXX M8 III M4 V       
+1104                      BAC6001-9 Fl Di(Human)                 {  0 } (C02+1) [0000] - -  - 012 11 NaXX K0 III       
```

Comment:
1104; The "Di(Human)" code says this world is the Human homeworld and they have died out here. The general "Di" code says there were some Sophonts here, but they're all gone now. I don't think we have a way of saying: There used to be a non-native population of race X here, but they're gone now.

Reply:
We have used zero pop and non-zero TL to indicate that elsewhere, with just the `Di` code.

Also... zero Pop, zero Gov, but non-zero Law? That's... weird.

Reply:
I've changed it to be simply Di and removed the law code oddity